packages/trader/src/Modules/Trading/Components/Form/ContractType/__tests__/contract-type-info.spec.tsx

```diff
@@ -0,0 +1,283 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import Info from '../ContractTypeInfo/contract-type-info';
+import userEvent from '@testing-library/user-event';
+
+jest.mock('Assets/Trading/Categories/trade-categories-gif.jsx', () => jest.fn(() => 'TradeCategoriesGif'));
+jest.mock('Assets/Trading/Categories/trade-categories.jsx', () => jest.fn(() => 'TradeDescription'));
+jest.mock('../ContractTypeInfo/contract-type-glossary', () => jest.fn(() => 'TradeTypeGlossary'));
+
+const mocked_props: React.ComponentProps<typeof Info> = {
+    handleSelect: jest.fn(),
+    item: {
+        text: 'Multipliers',
+        value: 'multiplier',
+    },
+    list: [
```

Comment:
I'm not sure if there is need to mention all contract types for mocked props

Reply:
no need but test files don't become part of the bundle in production so I think its ok to leave it as is